src/styles/blocks/is-active.scss

```diff
@@ -0,0 +1,13 @@
+.is-active.menu__link {
+  color: $main-header-color;
+
+  &::after {
+    content: ' ';
+    position: absolute;
```

Comment:
The `::after` pseudo-element uses `position: absolute;`. Make sure the parent element of `.is-active.menu__link` has a `position` property set (e.g., `relative`) to ensure the pseudo-element is positioned correctly relative to its parent.